addons/common/XEH_postInit.sqf

```diff
@@ -350,14 +358,6 @@ addMissionEventHandler ["PlayerViewChanged", {
 [QGVAR(displayTextStructured), {_this call FUNC(displayTextStructured)}] call CBA_fnc_addEventHandler;
 [QGVAR(displayTextPicture), {_this call FUNC(displayTextPicture)}] call CBA_fnc_addEventHandler;
 
-["ace_unconscious", {
```

Comment:
Just had a quick look over this myself again, I think deleting this EH was a mistake and may cause bugs with loading unconscious units into vehicles. I don't have access to a PC capable of running ArmA at the moment though, so I can't verify that.